magicctapipe/scripts/lst1_magic/semi_automatic_scripts/coincident_events.py

```diff
@@ -0,0 +1,374 @@
+                    "ulimit -s unlimited\n",
+                    "ulimit -a\n\n",
+                    f"export INM={nightMAGIC}\n",
+                    f"export OUTPUTDIR={nightLST}\n",
+                    "SAMPLE_LIST=($(<$OUTPUTDIR/list_LST.txt))\n",
+                    "SAMPLE=${SAMPLE_LIST[${SLURM_ARRAY_TASK_ID}]}\n",
+                    "export LOG=$OUTPUTDIR/coincidence_${SLURM_ARRAY_TASK_ID}.log\n",
+                    f"conda run -n {env_name} lst1_magic_event_coincidence --input-file-lst $SAMPLE --input-dir-magic $INM --output-dir $OUTPUTDIR --config-file {target_dir}/config_coincidence.yaml >$LOG 2>&1",
+                ]
+                f.writelines(lines)
+
+
+def main():
+
+    """
+    Here we read the config file and call the functions defined above.
+    """
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--config-file",
+        "-c",
+        dest="config_file",
+        type=str,
+        default="./config_general.yaml",
+        help="Path to a configuration file",
+    )
+
+    args = parser.parse_args()
+    with open(
+        args.config_file, "rb"
+    ) as f:  # "rb" mode opens the file in binary format for reading
+        config = yaml.safe_load(f)
+
+    telescope_ids = list(config["mc_tel_ids"].values())
+    target_dir = str(
+        Path(config["directories"]["workspace_dir"])
+        / config["directories"]["target_name"]
+    )
+    NSB_match = config["general"]["NSB_matching"]
+    env_name = config["general"]["env_name"]
+    source = config["directories"]["target_name"]
+    LST_version = config["general"]["LST_version"]
+    print("***** Generating file config_coincidence.yaml...")
+    print("***** This file can be found in ", target_dir)
+    configfile_coincidence(telescope_ids, target_dir)
+    nsb = config["general"]["nsb"]
+    runs_all = config["general"]["LST_runs"]
+    date = np.genfromtxt(runs_all, dtype=str, delimiter=",")
+    if not NSB_match:
+        nsblvl = 0
+        LST_runs_and_dates = config["general"]["LST_runs"]
+        LST_runs = np.genfromtxt(LST_runs_and_dates, dtype=str, delimiter=",")
+        print("***** Linking the paths to LST data files...")
+
+        print("***** Generating the bashscript...")
+        linking_bash_lst(
+            target_dir, LST_runs, nsblvl, date, source, LST_version, env_name, NSB_match
+        )  # linking the data paths to the current working directory
+
+        print("***** Submitting processess to the cluster...")
+        print(f"Process name: {target_dir.split('/')[-2:][1]}_coincidence")
+        print(
+            f"To check the jobs submitted to the cluster, type: squeue -n {target_dir.split('/')[-2:][1]}_coincidence"
+        )
+
+        # Below we run the bash scripts to find the coincident events
+        list_of_coincidence_scripts = np.sort(glob.glob("LST_coincident*.sh"))
+
+        for n, run in enumerate(list_of_coincidence_scripts):
+            if n == 0:
```

Comment:
how about this instead of if...else
```
launch_jobs = ""
for n, run in enumerate(list_of_coincidence_scripts):
    launch_jobs += (" && " if n>0 else "") +f"coincidence{n}=$(sbatch --parsable {run})"
````